components/gallery/GalleryItemPreviewer.vue

```diff
@@ -50,6 +50,7 @@ const isFullscreen = useVModel(props, 'value', emit)
 
 .gallery-item-modal {
   position: fixed;
+  margin-top: 5.25rem !important;
 
   :deep(&-content) {
```

Comment:
@prachi00 maybe change this line to this help fix the issue?

```suggestion
  :deep(.gallery-item-modal) {
```

Reply:
you are right, thanks